### ISSUE TITLE
Zoomable not working in Timeline

### ISSUE THREAD

**Issue by Kuurusch** [almende/vis#4092]
Because rollingMode is not working in the last release, i tried de develop-branch, because there it seems, that it is fixed. And yes, it works. What I'm facing now is, that zoomable seems not to work anymore. 
Any suggestions? Is it a bugg? Or do I miss something in the develop-version, which is new compared to the last release?

**Comment by skandalfo** [comment]
I've seen the same with the develop branch in Linux with Firefox Nightly (63.0a1 (2018-08-25) (64-bit)). The mouse wheel doesn't zoom.

It works with Chrome in the same environment.

**Comment by Kuurusch** [comment]
Yes, I've forgott to say, I'm working with Firefox 61.0.1, sorry!

**Comment by yotamberk** [comment]
Please try my new branch: 
I've now branched out to a new fork of my own including only Timeline.
https://github.com/yotamberk/timeline-plus
https://www.npmjs.com/package/timeline-plus
I maintain it weekly and new versions are up with fixes.

**Comment by Kuurusch** [comment]
@yotamberk  Ok thanks a lot, will test it...
An other question (sorry a little bit offtopic): Do you know, if the timeline optimizes optimizes element-updates when using the flush-functionality for the vis dataSet, such that it redraws only the last change made in the same item before calling flush()?

**Comment by yotamberk** [comment]
Ummm, not sure at all... You'll have to check it out
